configurations/configurations_test.go

```diff
@@ -18,6 +18,11 @@ CMD ["./code-runner/code-runner-server", "-port", "8082"]
 
 `
 
+var expectedAssgnEnvOutput = `FROM assignmentexec/trial2:2.0
```

Comment:
change to `expectedAssgnEnvDockerfileContents`.

Reply:
Okay